docs/source/models/causes/opioid_use_disorder/index.rst

```diff
@@ -0,0 +1,821 @@
+
+Assumptions and Limitations
+----------------------------
+
+**Key Assumptions**
+
+1. **Three-State Model**: The model simplifies the complex heterogeneity of OUD into three discrete states. In reality, OUD severity exists on a continuum, and individuals may have varying levels of use, dependence, and treatment engagement.
+
+2. **MOUD as Single Treatment State**: The "on_treatment" state aggregates all forms of MOUD (methadone, buprenorphine, naltrexone) despite important differences in effectiveness, retention, and accessibility across these medications.
+
+3. **No Differential Opioid Types**: The model does not distinguish between different opioid types (prescription vs. illicit; heroin vs. fentanyl) or routes of administration (oral vs. injection), which have different risk profiles.
+
+4. **Treatment Effectiveness**: The model assumes that MOUD substantially reduces disability and mortality, potentially to zero in the base case. Actual treatment effectiveness varies by medication type, dosage, adherence, and individual factors.
+
+5. **Steady-State Assumption**: Parameter estimation using DisMod-AT assumes approximate steady state or smooth temporal trends. Rapid changes in the opioid epidemic (e.g., fentanyl emergence) may violate this assumption.
+
+6. **Natural Remission**: The natural remission rate (*r*) represents spontaneous recovery without formal treatment. This is poorly characterized in the literature and is primarily identified through model calibration to match observed prevalence.
+
+**Limitations**
+
+1. **Treatment Coverage Data**: Estimates of the proportion of individuals with OUD receiving MOUD are highly uncertain and vary considerably across settings, populations, and data sources.
+
+2. **Remission Measurement**: Natural remission from OUD is difficult to measure directly, as individuals who recover without treatment are unlikely to be captured in treatment or surveillance systems.
+
+3. **Relapse Dynamics**: The model does not explicitly capture multiple cycles of treatment and relapse, which are common in OUD. Individuals may transition multiple times between states.
+
+4. **Comorbidities**: The model does not explicitly represent common comorbidities (mental health disorders, polysubstance use, infectious diseases) that influence OUD progression and treatment outcomes.
+
+5. **Heterogeneity**: Important sources of heterogeneity (age of onset, severity, social determinants, genetic factors) are not explicitly represented in the basic three-state model.
+
+6. **Overdose Mortality**: While the model includes excess mortality associated with OUD, overdose deaths may be coded separately in mortality data and require careful reconciliation with OUD-attributed deaths.
+
+7. **Intervention Effects**: Estimating the effect of specific interventions on transition rates (e.g., how much does a particular treatment expansion program increase *ti*?) requires additional assumptions and calibration.
```

Comment:
This seems like not a limitation, but merely the fact that interventions haven't been documented yet.